PortalFetch/crawler_sel.py

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/env python
+"""Fetch course information from De Anza myportal.
+
+It requires the users input their own user name and password.
+"""
+from configparser import ConfigParser
+from selenium import webdriver
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import Select
+from selenium.common.exceptions import NoSuchElementException
+import sys
+import csv
+from bs4 import BeautifulSoup
+import time
+import chromedriver_binary
+import logging
```

Comment:
logging is imported, but need to set the `basicConfig`  for the location, level, format of the logger.
### [Reference](https://cuiqingcai.com/6080.html)